lib/provider/direction_notifier.dart

```diff
@@ -2,14 +2,23 @@ import 'package:concordi_around/model/coordinate.dart';
 import 'package:concordi_around/model/direction.dart';
 import 'package:concordi_around/service/map_constant.dart';
 import 'package:concordi_around/service/map_direction.dart';
+import 'package:concordi_around/service/map_helper.dart';
 import 'package:flutter/cupertino.dart';
```

Comment:
This wasn't your addition and is already in master, but typically the import should be flutter/material.dart for consistency sake. But to further elaborate on this, flutter/material.dart and flutter/cupertino.dart packages are higher level layers that reexport many of the same packages. In this case, you actually only need the flutter/foundation.dart package to use ChangeNotifier. If you wanted to correct this for precision purposes by all means, else this does not have anything to do with the approval of your additions.